src/Sarif.UnitTests/TestData/PrereleaseCompatibilityTransformer/Inputs/RunResources.sarif

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2018-10-10",
+  "version": "2.0.0-csd.2.beta.2018-10-10",
+  "runs": [
+    {
+      "tool": {
+        "name": "CodeScanner",
+        "semanticVersion": "2.1.0"
+      },
+      "resources": {
+        "messageStrings": {"noContent": "This file has no content to review."},
+        "rules": {
+          "SHARED-ID.2": {
+            "id": "SHARED-ID",
+            "configuration": {
+              "defaultLevel": "error",
+              "enabled": false
```

Comment:
>false [](start = 25, length = 5)

This rule is disabled by default, yet there is a `result` that refers to it. Just for the sake of making this beautiful example self-consistent, there should be an `invocation` object whose `ruleConfiguration` enables this rule.

Reply:
yes! this data is weird. but we don't have a way to express the whole world. in the new world, we have a default enabled state _and_ a rule configuration on the invocation that may diverge. I added enabled == false here to get a non-default value for the bool. to make the example consistent, I need to add a new rule that is disabled (with no corresponding output). Will do that.

---
In reply to: [255309560](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255309560) [](ancestors = 255309560)